src/dev-server/client/app-error.ts

```diff
@@ -199,12 +205,12 @@ const getDevServerModal = (doc: Document) => {
 
   outer.innerHTML = `<style>${appErrorCss}</style><div id="${DEV_SERVER_MODAL}-inner"></div>`;
 
-  return doc.getElementById(`${DEV_SERVER_MODAL}-inner`);
+  return doc.getElementById(`${DEV_SERVER_MODAL}-inner`) as HTMLElement;
 };
 
 export const clearAppErrorModal = (data: { window: Window }) => {
   const appErrorElm = data.window.document.getElementById(DEV_SERVER_MODAL);
-  if (appErrorElm) {
+  if (appErrorElm && appErrorElm.parentNode) {
```

Comment:
Not necessary, but could be coalesced
```suggestion
  if (appErrorElm?.parentNode) {
```

Reply:
yea more concise!